public/app/screens/main/sections/device-list/device-finder/device-finder.jsx

```diff
@@ -65,26 +91,35 @@ var DeviceFinder = React.createClass({
       action={this.startSearching} />;
   },
 
+  getDeviceFoundDialog: function getNoResultsDialog () {
+    var status = <div className='cc-device-finder__status-warning'>Device found!</div>;
+
+    return <DeviceFinderDialog
+      status={status}
+      showSeparator='true'
+      title='Searching for new devices'
+      description='Please make sure that all devices are in discovery mode'
+      closeAction={this.closeDialog}
+      actionText='Ok'
+      action={this.showFoundDeviceInfo} />;
+  },
+
   startSearching: function startSearching () {
-      this.setState({ activity: 'searching' });
+    this.setState({activity: DeviceFinderActivity.SEARCHING});
```

Comment:
How did you pass the style check ?
There should be whitespaces between `{` `}` and values.